jsonnet/components/prometheus.libsonnet

```diff
@@ -203,7 +203,25 @@ function(params)
       data: {},
     },
 
-    kubeRbacProxySecret: generateSecret.staticAuthSecret(cfg.namespace, cfg.commonLabels, 'kube-rbac-proxy'),
+    kubeRbacProxySecret: generateSecret.staticAuthSecret(
+      cfg.namespace,
+      cfg.commonLabels,
+      'kube-rbac-proxy',
+      {
+        authorization+: {
+          static+: [
+            {
+              user: {
+                name: 'system:serviceaccount:openshift-monitoring:prometheus-k8s',
```

Comment:
nit: This seems to repeat at various place. Perhaps we should store this in a variable / config.

Reply:
the other reference to this user is here https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/utils/generate-secret.libsonnet#L18

I think it not so repetitive as all the `*rbac-secret.yaml` files are generated from that function.

Reply:
if its ok for @sthaha we can moving forward with this PR and maybe do a refactor in the future with other variables configs or SA.

Reply:
@marioferh , `nit` can always be ignored 🤗